src/runtime/start/start-module.ts

```diff
@@ -78,7 +78,7 @@ export function createStartModuleContent(config: StartModuleConfig): string {
       require('${
         config.absoluteModuleImports
           ? config.layout.packageJson.path
-          : Path.relative(config.layout.buildOutputRelative, config.layout.packageJson.path)
+          : Path.relative(config.layout.buildOutput, config.layout.packageJson.path)
```

Comment:
There's another fix here, the biggest reason why I wanted to use an absolute path across the codebase for the build output. It just makes everything safer and prevents us from joining the relative output build to something that doesn't make sense